sdk/textanalytics/Azure.AI.TextAnalytics/src/TasksStateTasks.cs

```diff
@@ -7,15 +7,15 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// TasksStateTasks.
+    /// AnalyzeTasks.
     /// </summary>
     [CodeGenModel("TasksStateTasks")]
-    public partial class TasksStateTasks
+    public partial class AnalyzeTasks
```

Comment:
could you remind me again why the need of this class to be customizable? besides the fact that it needs to be public?

Reply:
UPdating the name from TasksStateTasks to AnalyzeTasks and moreover passing idToMapIndex to results.